apps/aecore/src/aec_parent_connector.erl

```diff
@@ -204,6 +225,28 @@ handle_call(get_parent_conn_mod, _From, #state{parent_conn_mod = Mod} = State) -
 handle_call(get_parent_chain_type, _From, #state{parent_conn_mod = Mod} = State) ->
     Reply = Mod:get_chain_type(),
     {reply, Reply, State};
+handle_call(get_pinning_data, _From, #state{parent_conn_mod = Mod, network_id = NetworkID} = State) ->
+    Reply = get_pinning_data_(Mod, NetworkID),
+    {reply, Reply, State};
+handle_call({pin_to_pc, Who, Amount, Fee}, _From, #state{parent_conn_mod = Mod, parent_hosts = ParentHosts, hcpc = HCPCMap, sign_module = SignModule, network_id = NetworkID} = State) ->
+    Reply = case maps:is_key(Who, HCPCMap) of
+        true ->
+            {ok, PinningData} = get_pinning_data_(Mod, NetworkID),
+            handle_parent_pin_calls(Mod, pin_to_pc, {PinningData, maps:get(Who, HCPCMap), Amount, Fee, NetworkID, SignModule}, ParentHosts);
+        false -> {error, {no_pc_account_for_staker, Who}}
+    end,
+    {reply, Reply, State};
+handle_call({pin_tx_to_cc, PinTx, Who, Amount, Fee}, _From, #state{parent_conn_mod = Mod, sign_module = SignModule} = State) ->
+    lager:debug("In here?",[]),
```

Comment:
😅

Reply:
Oops will clean up such stuff as part of refac.